site/src/docs/components/koros/index.mdx

```diff
@@ -30,6 +30,9 @@ export default ({ children, pageContext }) => <TabsLayout pageContext={pageConte
 - Use koros as part of a background. Do not create new visual elements using koros.
 - It is allowed to animate Koros shapes (not included in the HDS yet).
 - A koro shape can be rotated in 45 degree increments. For more information see the [Angled koro variant documentation](#angled) below.
+- Koros should be centered and have at least 2 full wave motif tips visible
+- Do not use Koro shapes as an accent element (not for example, for active element)
```

Comment:
- Koro shapes is in plural, so it should be: "--Koro shapes as accent elements--". 
- The end of sentence might "flow" better if restructured like this: "- Do not use Koro shapes as accent elements, for example to display active elements."
- Remember a dot at the end